Mlem/Views/Tabs/Feeds/Feed Types/AggregateFeedView.swift

```diff
@@ -0,0 +1,169 @@
+                    } catch {
+                        errorHandler.handle(error)
+                    }
+                }.value
+            }
+            .background {
+                Color.systemBackground
+            }
+            .fancyTabScrollCompatible()
+            .toolbar {
+                ToolbarItem(placement: .principal) {
+                    navBarTitle
+                        .opacity(scrollToTopAppeared ? 0 : 1)
+                        .animation(.easeOut(duration: 0.2), value: scrollToTopAppeared)
+                }
+            }
+            .navigationBarTitleDisplayMode(.inline)
+            .navigationBarColor(visibility: .automatic)
+    }
+    
+    @ViewBuilder
+    var content: some View {
+        VStack(spacing: 0) {
+            ScrollView {
+                VStack(spacing: 0) {
+                    VStack(spacing: 0) {
+                        ScrollToView(appeared: $scrollToTopAppeared)
+                            .id(scrollToTop)
+                        headerView
+                            .padding(.top, -1)
+                    }
+                    
+                    PostFeedView(postSortType: $postSortType, showCommunity: true)
+                        .environmentObject(postTracker)
+                }
+            }
+        }
+    }
+    
+    @ViewBuilder
+    var headerView: some View {
+        Menu {
+            ForEach(genFeedSwitchingFunctions()) { menuFunction in
+                MenuButton(menuFunction: menuFunction, confirmDestructive: nil)
+            }
+        } label: {
+            VStack(spacing: 0) {
+                HStack(alignment: .center, spacing: AppConstants.postAndCommentSpacing) {
+                    Image(systemName: postTracker.feedType.iconNameCircle)
+                        .resizable()
+                        .frame(width: 44, height: 44)
+                        .foregroundStyle(postTracker.feedType.color ?? .primary)
+                        .padding(.leading, AppConstants.postAndCommentSpacing)
+                        
+                    VStack(alignment: .leading, spacing: 0) {
+                        HStack(spacing: 5) {
+                            Text(postTracker.feedType.label)
+                                .lineLimit(1)
+                                .minimumScaleFactor(0.01)
+                                .fontWeight(.semibold)
+                            Image(systemName: Icons.dropdown)
+                                .foregroundStyle(.secondary)
+                        }
+                        .font(.title2)
+                            
+                        Text(subtitle)
+                            .font(.footnote)
+                            .foregroundStyle(.secondary)
+                    }
+                    .frame(height: 44)
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                }
+                .padding(.vertical, 5)
+                    
```

Comment:
Can we pad this a little more to make it consistent with the community view? 
```suggestion
                        .padding(.bottom, 3)
```